test/Tests.Infrastructure/RavenTestHelper.cs

```diff
@@ -37,7 +37,7 @@ public static string NewDataPath(string testName, int serverPort, bool forceCrea
         {
             testName = testName?.Replace("<", "").Replace(">", "");
 
-            var newDataDir = Path.GetFullPath($".\\Databases\\{testName ?? "TestDatabase"}_{serverPort}-{DateTime.Now:yyyy-MM-dd-HH-mm-ss-fff}-{Interlocked.Increment(ref _pathCount)}");
+            var newDataDir = Path.GetFullPath($".\\Databases\\{testName ?? "TestDatabase"}@{serverPort}-{Interlocked.Increment(ref _pathCount)}");
```

Comment:
@ayende the '@' char is causing some test failures. Can we switch back to '_' ?

Reply:
Yes, we can switch it back, I'll do so shortly.